internal/configs/version1/nginx-plus.tmpl

```diff
@@ -70,6 +79,13 @@ http {
     {{if .AppProtectPhysicalMemoryThresholds}}app_protect_physical_memory_util_thresholds {{.AppProtectPhysicalMemoryThresholds}};{{end}}
     include /etc/nginx/waf/nac-usersigs/index.conf;
     {{- end}}
+
+    {{- if .AppProtectDosLoadModule}}
+        {{- if .AppProtectDosLivenessEnable}}
+    app_protect_dos_liveness {{.AppProtectDosLivenessEnable}} uri:/{{.AppProtectDosLivenessURI}} port:{{.AppProtectDosLivenessPort}};
```

Comment:
Are these values (`AppProtectDosLivenessURI` and `AppProtectDosLivenessPort`) optional or required? 
If they are required, we can help the user by validating they are not empty when parsing the config map in `configmaps.go`